src/System.Data.Common/pkg/System.Data.Common.pkgproj

```diff
@@ -3,9 +3,11 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Data.Common.csproj">
-      <SupportedFramework>net451;netcore451;netstandardapp1.5;wpa81</SupportedFramework>
+      <SupportedFramework>net451;netcore451;netstandardapp1.5;wp8;wpa81</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\src\System.Data.Common.builds" />
+    <ExcludeDefaultValidateFramework Include="net45" />
```

Comment:
You shouldn't exclude.  It should be validating that the package is **not supported** on those frameworks.  If its giving you errors that may mean you are packaging in a way that the package might be supporting those incorrectly.